internal/commands/pull/inputs.go

```diff
@@ -0,0 +1,118 @@
+package pull
+
+import (
+	"errors"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/mitchellh/go-homedir"
+)
+
+const (
+	flagFrom      = "from"
+	flagFromShort = "a"
+	flagFromUsage = "specify the app to pull changes down from"
+
+	flagProject      = "project"
+	flagProjectUsage = "the MongoDB cloud project id"
+
+	flagAppVersion      = "app-version"
+	flagAppVersionUsage = "specity the app config version to pull changes down as"
+
+	flagTarget      = "target"
+	flagTargetShort = "t"
+	flagTargetUsage = "provide the path to export a Realm app to"
+
+	flagIncludeDependencies      = "include-dependencies"
+	flagIncludeDependenciesShort = "d"
+	flagIncludeDependenciesUsage = "specify the command to push Realm app dependencies changes as well"
+
+	flagIncludeHosting      = "include-hosting"
+	flagIncludeHostingShort = "s"
+	flagIncludeHostingUsage = "specify the command to push Realm app hosting changes as well"
+
+	flagDryRun      = "dry-run"
+	flagDryRunShort = "x"
+	flagDryRunUsage = "specify the command to run without writing any changes to the file system"
```

Comment:
[q] These flags all start with `specify the command to` even though they are bools. Is there something missing or something I don't understand.

Reply:
you're not missing anything, im just bad with words.  i won't worry too much about the particular language in the flags' usages just yet, but the intention here was that a `bool` flag is used to "specify the command to" run/push/do stuff

Reply:
KO, just to have my thoughts put down, I think the language around bool flags might be better suited to be `include to <behavior>`

Reply:
i like that, updated the `bool` flags for `push` and `pull` here